### PR TITLE
(SIMP-3457) Update Journald

### PR DESCRIPTION
**Pull request by jeannegreulich**
- updated puppet file
  - updated simp extras in simp build file

SIMP-3457 #close
SIMP-3612 #close

### CODE REVIEW

File: Puppetfile.tracking [hunk 1]
```diff
@@ -252,7 +252,7 @@ mod 'simp-jenkins',
   # SIMP has made changes to this module that don't exist upstream
```
Comment (lnemsick-simp) [inline Puppetfile.tracking]: Please remove the comment about 'SIMP has made changes'

File: Puppetfile.tracking [hunk 2]
```diff
@@ -252,7 +252,7 @@ mod 'simp-jenkins',
...
 mod 'simp-journald',
   :git => 'https://github.com/simp/pupmod-simp-journald.git',
-  :ref => 'simp-master'
+  :ref => 'master'
```
Comment (lnemsick-simp) [inline Puppetfile.tracking]: Please pin the version to a tag that we want to use?

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 3]
```diff
@@ -186,7 +186,7 @@ rubygem-puppet-lint:
   :rpm_name: rubygem-puppet-lint-1.1.0-1.el7.noarch.rpm
   :source: http://yum.puppetlabs.com/el/7/dependencies/x86_64/rubygem-puppet-lint-1.1.0-1.el7.noarch.rpm
 rubygem-rake:
-  :rpm_name: rubygem-rake-0.9.6-25.el7_1.noarch.rpm
+  :rpm_name: rubygem-rake-0.9.6-29.el7.noarch.rpm
   :source: http://mirrors.advancedhosters.com/centos/7.3.1611/os/x86_64/Packages/rubygem-rake-0.9.6-29.el7.noarch.rpm
```
Comment (lnemsick-simp) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: good catch!

Comment (jeannegreulich) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Made the changed requested.

Comment (jeannegreulich) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Made the changes requested.

**Pull request merged**